python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -93,8 +97,10 @@ def start_ganga():
     logger.info("Checking Credentials")
 
     if missing_cred:
-        raise Exception("Failed due to missing credentials %s" %
-                        str(missing_cred))
+        raise Exception("Failed due to missing credentials %s" % str(missing_cred))
+
+    from Ganga.GPI import queues
+    queues.unlock()
```

Comment:
Is it possible to access the queues object directly and not through the GPI here?